templates/20years/index.html

```diff
@@ -0,0 +1,1210 @@
+          <div class="col-3">
+            <p class="p-heading--5">
+              <a href="https://events.canonical.com/event/51/">Join us at the Ubuntu Summit 2024</a>
+            </p>
+          </div>
+          <div class="col-6">
+            <p>
+              Taking place from 25-27 October in The Hague, the Netherlands, this summit is where we bring together experts, builders, tinkerers and engineers.
+            </p>
+            <p>Join us to drive forward our collective mission to build the future with open source software.</p>
+          </div>
+        </div>
+        <hr class="is-muted" />
+        <div class="row">
+          <div class="col-3">
+            <p class="p-heading--5">
+              <a href="/community/contribute">Become a member of the community</a>
+            </p>
+          </div>
+          <div class="col-6">
+            <p>
+              Ubuntu is most of all a community. All of the software, artwork and documentation in Ubuntu has been created, tested, used and discussed openly by people around the world. Anyone who uses Ubuntu is part of this global community, and we invite you to help shape Ubuntu to better meet your needs. To make it yours!
+            </p>
+          </div>
+        </div>
+        <hr class="is-muted" />
+        <div class="row">
+          <div class="col-3">
+            <p class="p-heading--5">
+              <a href="https://canonical.com/careers/all">Be a part of the story by joining Canonical</a>
+            </p>
+          </div>
+          <div class="col-6">
+            <p>
+              We can't do what we do without the amazing people that work with us. If you're interested in starting your next chapter with us, take a look at our careers page.
+            </p>
+          </div>
+        </div>
+        <hr class="is-muted" />
+        <div class="row">
+          <div class="col-3">
+            <p class="p-heading--5">
+              <a href="https://canonical.com/blog/canonical-offers-12-year-lts-for-any-open-source-docker-image">Everything LTS: securing open source beyond Ubuntu</a>
+            </p>
+          </div>
+          <div class="col-6">
+            <p>
+              With Everything LTS, we're breaking new ground and offering up to 12 years of long term support for any open source docker image. We're taking the security maintenance commitments we've made to Ubuntu to the wider world of open source.
+            </p>
+            <p>
+              That means CVE patching for upstream components in your entire open source dependency tree, whether you host your services on Ubuntu, RHEL, VMware or public cloud K8s.
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <hr class="p-rule is-fixed-width" />
+  <section class="p-strip is-deep">
+    <div class="u-fixed-width">
+      <h2>
+        <a class="js-invoke-modal" href="/contact-us">Get notified when Ubuntu 24.10 launches&nbsp;&rsaquo;</a>
```

Comment:
This leads to a standard https://ubuntu.com/contact-us. And there is no form to sign up for newsletters.

Reply:
Hey, thanks for flagging this. I am aware, just waiting on feedback from the pdm.

This PR isn't ready for dev code&qa yet, so there is likely a number of issues like this.